### DISCUSSION TITLE
Multi Hosts - Merged

### DISCUSSION THREAD

**Discussion by matthiscock** [opcodesio/log-viewer#222] [Q&A] [answered]
Log viewer is excellent, but I wonder if this is possible.
We manage several laravel sites.
We would love have it so that we have 'one marged list' of log events for all the hosts we have setup - rather than having to choose one.

Similar to a unified inbox on popular inbox clients

This way we can react to any logs (or errors) we see on any of the sites we manage

Is there a way of doing this?

**Comment by arukompas** [comment, marked as answer]
Hey @matthiscock great question, but I don't think the Log Viewer is, or ever will be, as sophisticated as that.

For this particular use case I definitely recommend a log aggregation service that stores the logs in the cloud and can give better insights into what's happening where.

For now, the Log Viewer can display logs from a single host at a time only.
